packages/labs/package.json

```diff
@@ -11,9 +11,24 @@
     "bugs": {
         "url": "https://github.com/bazelbuild/rules_nodejs/issues"
     },
-    "devDependencies": {},
-    "dependencies": {},
+    "devDependencies": {
+        "@bazel/typescript": "^0.25.1",
```

Comment:
Should probably be `"@bazel/typescript": "bazel://@npm_bazel_typescript//:npm_package",` like /internal/karma/package.json

Reply:
done